### PR TITLE
Week 1 Day 3 post and banner

### PR DESCRIPTION
**Pull request by smurawski**

### CODE REVIEW

File: website/blog-cnny/2023-01-25/30days.md [hunk 1]
```diff
@@ -23,37 +23,78 @@ tags: [cloud-native, 30daysofcloudnative, zero-to-hero, ask-the-expert, azure-ku
...
+:::
 
-## Resources
\ No newline at end of file
+## Introduction
+
+Kubernetes is an open source container orchestration engine that can help with automated deployment, scaling, and management of our applications.
+
+Kubernetes takes physical (or virtual) resources and provides a consistent API over them, bringing a consistency to the management and runtime experience for our applications.  Kubernetes provides us with a number of capabilities such as:
+
```
Comment (pauldotyu) [inline website/blog-cnny/2023-01-25/30days.md:68]: Should we include "Container scheduling" here as well?

File: website/blog-cnny/2023-01-25/30days.md [hunk 2]
```diff
@@ -23,37 +23,78 @@ tags: [cloud-native, 30daysofcloudnative, zero-to-hero, ask-the-expert, azure-ku
...
+* Automated rollouts and rollbacks
+* Automatic bin packing
+* Self-healing
+* Secret and configuration management
+
+We'll learn more about most of these topics as we progress through Cloud Native New Year.
+
+## What is Kubernetes?
+
+Let's hear from Brendan Burns, one of the creators of Kubernetes as to what Kubernetes actually is.
```
Comment (pauldotyu) [inline website/blog-cnny/2023-01-25/30days.md:80]:
```suggestion
Let's hear from Brendan Burns, one of the founders of Kubernetes as to what Kubernetes actually is.
```

Comment (pauldotyu) [reply website/blog-cnny/2023-01-25/30days.md:80]: just to keep the "title" consistent with the video

File: website/blog-cnny/2023-01-25/30days.md [hunk 3]
```diff
@@ -23,37 +23,78 @@ tags: [cloud-native, 30daysofcloudnative, zero-to-hero, ask-the-expert, azure-ku
...
+
+And Brendan shares a bit more with us about how Kubernetes works.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/daVUONZqn88" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+## Conclusion
+
+Kubernetes allows us to deploy and manage our applications effectively and consistently. 
+
+By providing a consistent API across many of the concerns our applications have, like load balancers, networking, storage, and compute, Kubernetes improves both our ability to build new software.
```
Comment (pauldotyu) [inline website/blog-cnny/2023-01-25/30days.md:94]:
```suggestion
By providing a consistent API across many of the concerns our applications have, like load balancing, networking, storage, and compute, Kubernetes improves both our ability to build and ship new software. 
```

**Review by pauldotyu — COMMENTED** [review]
Good stuff! Love the inclusion of some of Brendan's videos here!

File: website/blog-cnny/2023-01-25/30days.md (hunk 1 above)
Comment (smurawski) [reply website/blog-cnny/2023-01-25/30days.md:68]: Done

**Pull request closed without merging**